src/main/frontend/fs.cljs

```diff
@@ -201,7 +201,10 @@
       (fn [_stat])
       (fn [_error]
         (mkdir! dir))))
-   (p/catch (fn [error] (js/console.error error)))))
+   (p/catch (fn [error]
```

Comment:
This error code is related to the fs implementation. It would be better to add it to fs/node.cljs

Reply:
@andelf I assume you mean by modifying the `mkdir` function in node.cljs? 

I am a bit confused since it seems like all errors are not handled in node.cljs and they are handled in the caller functions in fs.cljs.

I am not sure if the call to `util/p-handle` should be in fs.cljs or node.cljs.

I have attempted a fix in the latest commit.